### PR TITLE
Initial load of Jefferson's Episerver + C# post

### PR DESCRIPTION
**Pull request by Quorralyne**
This blog post has the following features:

- [X] A GitHub Repository with a polished README
- [X] A title that's approved by Dev Marketing
- [X] A URL that's approved by Dev Marketing
- [X] Rendered locally and confirmed that no Markdown typos exist

### CODE REVIEW

File: _source/_posts/2019-08-15-episerver-csharp-aspnet-cms.md [hunk 1]
```diff
@@ -0,0 +1,322 @@
...
+
+In this post, we will integrate [Episerver CMS](https://www.episerver.com/) with Okta via OpenID Connect, resulting in a centralized user identity store for your e-commerce customers through the Okta Identity Cloud platform.
+
+>For this project, you'll need to prepare a few dependencies:
+>1. [Visual Studio 2019](https://visualstudio.microsoft.com/vs/)
+>2. [Episerver Visual Studio Extension](https://marketplace.visualstudio.com/items?itemName=Episerver.EpiserverCMSVisualStudioExtension)
+>3. Microsoft NuGet add-on libraries
+>4. A free [Okta Developer](https://developer.okta.com/) account
+
+## Install Visual Studio to Build Episerver Website
```
Comment (reverentgeek) [inline _source/_posts/2019-08-15-episerver-csharp-aspnet-cms.md]: I recommend getting the `markdownlint` extension for VS Code. There are a number of warnings in this file, such as not having a blank line around a few of the subheadings, and some bare URLs.

File: _source/_posts/2019-08-15-episerver-csharp-aspnet-cms.md [hunk 2]
```diff
@@ -0,0 +1,322 @@
...
+First things first, you need to download and install [Visual Studio](https://visualstudio.microsoft.com/vs/). The community edition is perfectly fine for what we are going to do.
+
+
```
Comment (reverentgeek) [inline _source/_posts/2019-08-15-episerver-csharp-aspnet-cms.md]: Remove extra blank line

File: _source/_posts/2019-08-15-episerver-csharp-aspnet-cms.md [hunk 3]
```diff
@@ -0,0 +1,322 @@
...
+{% img blog/episerver/VS2019getstarted.png alt:"Visual Studio get started" width:"800" %}{: .center-image }
+## Install Episerver CMS Visual Studio Extension
```
Comment (reverentgeek) [inline _source/_posts/2019-08-15-episerver-csharp-aspnet-cms.md]: Add blank lines around this heading

File: _source/_posts/2019-08-15-episerver-csharp-aspnet-cms.md [hunk 4]
```diff
@@ -0,0 +1,322 @@
...
+There are two ways to install the extension. The first  is via the link above and the other one is through the Visual Studio IDE, demonstrated below.
+
+On the Visual Studio 2019 landing screen shown above, click **Continue without code**.
+On the Extensions Menu bar, click **Manage Extensions**. Finally, click **Online** and in the search bar, query for *Episerver*.
+
+
+{% img blog/episerver/VS2019episerverextension.png alt:"Visual Studio Episerver extension" width:"800" %}{: .center-image }
+
+Install the extension and once done, restart Visual Studio.
+## Create an Episerver CMS Template Project (Alloy)
```
Comment (reverentgeek) [inline _source/_posts/2019-08-15-episerver-csharp-aspnet-cms.md]: Add blank lines around heading

File: _source/_posts/2019-08-15-episerver-csharp-aspnet-cms.md [hunk 5]
```diff
@@ -0,0 +1,322 @@
...
+                }
+            });
+        }
+    }
+}
+```
+
+Remove the entire `public void Configuration(IAppBuilder app)` and replace the content with this code on GitHub.
+
+Modified *Startup.cs* in github: https://github.com/hawjefferson/EpiserverCMSOkta/blob/master/Startup.cs
```
Comment (reverentgeek) [inline _source/_posts/2019-08-15-episerver-csharp-aspnet-cms.md]: no bare URLs

File: _source/_posts/2019-08-15-episerver-csharp-aspnet-cms.md [hunk 6]
```diff
@@ -0,0 +1,322 @@
...
+    /// <summary>
+    /// How long the web application authentication cookie is valid (in minutes in our example).
+    /// </summary>
+    public const int AuthCookieValidMinutes = 60;
+}
+```
+
+## Setup Okta for Authentication in Episerver
+
+If you haven't already, head on over to https://developer.okta.com/signup/ and create a free Okta org.
```
Comment (reverentgeek) [inline _source/_posts/2019-08-15-episerver-csharp-aspnet-cms.md]: No bare URLs

File: _source/_posts/2019-08-15-episerver-csharp-aspnet-cms.md [hunk 7]
```diff
@@ -0,0 +1,322 @@
...
+
+For this example, I'll be using my own Okta tenant: `https://identity.hawservers.com`. This tenant has been configured to use a vanity URL, which you can also setup by following [this guide](https://help.okta.com/en/prod/Content/Topics/Settings/custom-url-domain.htm).
+
+Go ahead and login to your Okta account, click **Applications** and then **Add Application**. Next, click **Web** and then **Next**.
+
+This creates an OpenID Connect application representing your Episerver application.
+
+{% img blog/episerver/oktaappconfig.png alt:"Okta application configuration" width:"1000" %}{: .center-image }
+
+You'll be presented with a screen to configure your Okta OpenID Connect application where you can change the name to Episerver Application. As for the *Login redirect URIs* field, this will be the url when you run your Episerver CMS instance (e.g. http://localhost:{port}).
```
Comment (reverentgeek) [inline _source/_posts/2019-08-15-episerver-csharp-aspnet-cms.md]: Add backticks around `http://localhost:{port}`

File: _source/_posts/2019-08-15-episerver-csharp-aspnet-cms.md [hunk 8]
```diff
@@ -0,0 +1,322 @@
...
+
+Once authenticated, you should be redirected back to the home page.
+
+{% img blog/episerver/episerverauthenticatedpage.png alt:"Episerver authenticated home page" width:"1000" %}{: .center-image }
+
+If you check your console log in Visual Studio, you should see something like this:
+
+{% img blog/episerver/VS2019consolelog.png alt:"Visual Studio Episerver log output" width:"1000" %}{: .center-image }
+
+```
```
Comment (reverentgeek) [inline _source/_posts/2019-08-15-episerver-csharp-aspnet-cms.md]: needs a code block identifier

File: _source/_posts/2019-08-15-episerver-csharp-aspnet-cms.md [hunk 9]
```diff
@@ -0,0 +1,322 @@
...
+
+## Install Visual Studio to Build Episerver Website
+First things first, you need to download and install [Visual Studio](https://visualstudio.microsoft.com/vs/). The community edition is perfectly fine for what we are going to do.
+
+
+{% img blog/episerver/VS2019getstarted.png alt:"Visual Studio get started" width:"800" %}{: .center-image }
+## Install Episerver CMS Visual Studio Extension
+There are two ways to install the extension. The first  is via the link above and the other one is through the Visual Studio IDE, demonstrated below.
+
+On the Visual Studio 2019 landing screen shown above, click **Continue without code**.
```
Comment (reverentgeek) [inline _source/_posts/2019-08-15-episerver-csharp-aspnet-cms.md]: Need a blank line between these sentences, or they will be combined into one paragraph.

File: _source/_posts/2019-08-15-episerver-csharp-aspnet-cms.md (hunk 8 above)
Comment (Quorralyne) [reply _source/_posts/2019-08-15-episerver-csharp-aspnet-cms.md]: What would the identifier be for console output in markdown?

File: _source/_posts/2019-08-15-episerver-csharp-aspnet-cms.md (hunk 1 above)
Comment (Quorralyne) [reply _source/_posts/2019-08-15-episerver-csharp-aspnet-cms.md]: Ooo... Great suggestion!

File: _source/_posts/2019-08-15-episerver-csharp-aspnet-cms.md (hunk 8 above)
Comment (reverentgeek) [reply _source/_posts/2019-08-15-episerver-csharp-aspnet-cms.md]: usually `sh` or `bash`

**Pull request merged**